src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 0;
```

Comment:
Create a const variable for `days * 40`. You duplicate this value a lot
![image](https://user-images.githubusercontent.com/96409563/231249749-375c5ee4-4d87-422a-bf44-bfb96cf3927e.png)